packages/wrangler/src/dev/dev.tsx

```diff
@@ -441,7 +441,7 @@ function useCustomBuild(expectedEntry: Entry, build: Config["build"]): void {
 		}
 
 		return () => {
-			watcher?.close();
+			void watcher?.close();
```

Comment:
Why do we need this?

Reply:
Oh is it because it returns a promise and TS will complain about lost promise rejections?

Reply:
Well glad that is working then.